plugins/scaffolder-backend/src/scaffolder/tasks/StorageTaskBroker.ts

```diff
@@ -60,6 +62,16 @@ export class TaskManager implements TaskContext {
     return this.task.secrets;
   }
 
+  get abortContext() {
+    return {
+      abort: async () => {
+        this.ac.abort();
+        this.isAborted = true;
```

Comment:
No need for this; you already have `ac.signal.aborted` to use at any point. So I'd say, just return the raw AbortController from this method instead of wrapping it up, and skip the isAborted member, probably remove the `aborted` method below too then right since it just duplicates information?